apisix/core/ctx.lua

```diff
@@ -274,6 +274,15 @@ do
                     end
                 end
 
+            elseif core_str.has_prefix(key, "uri_arg_") then
```

Comment:
`uri_arg` always means URL query arguments, maybe you should use `uri_para_` instead?

Reply:
How about `uri_param`?

Reply:
LGTM